src/ast/mod.rs

```diff
@@ -424,7 +424,7 @@ pub enum CastKind {
     /// See <https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#safe_casting>.
     SafeCast,
     /// `<expr> :: <datatype>`
-    DoubleColon,
+    DoubleColonCast,
```

Comment:
Please rename this back to `DoubleColon`. The others have the word `Cast` because you actually use the word `CAST` in the syntax. This is also a breaking change without enough benefit to justify.

Reply:
agreed